src/courtroom.cpp

```diff
@@ -1347,6 +1347,13 @@ void Courtroom::on_chat_return_pressed()
       else
         f_emote_mod = 2;
     }
+    else
+    {
+      if (f_emote_mod == 1)
+        f_emote_mod = 0;
+      else if (f_emote_mod == 4)
+        f_emote_mod = 5;
+    }
```

Comment:
Seems contrived. Probably this if block should be condensed to `if (objection_state >0 && ui_pre->isChecked())`. Then the control flow can go to the `else` block at the very bottom.

Reply:
Done (002c940), good catch.